src/js/view/controlbar.js

```diff
@@ -302,23 +280,15 @@ define([
         onPlaylist : function(model, playlist) {
             var display = (playlist.length > 1);
             this.elements.next.toggle(display);
-            this.elements.prev.toggle(display);
-            if(this.elements.playlist) {
-                this.elements.playlist.setup(playlist, model.get('item'));
-            }
         },
-        onPlaylistItem : function(model/*, item*/) {
+        onPlaylistItem : function(/*model, item*/) {
```

Comment:
I think we should just remove these comments, or at least move them out of the signature